### PR TITLE
sql: add TiDB specific variables for 3.0

### PR DESCRIPTION
**Pull request by crazycs520**

### CODE REVIEW

File: dev/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 1]
```diff
@@ -57,6 +57,30 @@ set @@global.tidb_distsql_scan_concurrency = 10
 
 这个变量用来设置优化器是否执行 in-子查询展开的优化操作。
 
+### tidb_auto_analyze_ratio
+
+作用域: GLOBAL
+
+默认值: 0.5
+
+这个变量用来设置自动 ANALYZE 更新的阈值。当某个表 `tbl` 的修改行数与总行数的比值大于 tidb_auto_analyze_ratio，并且当前时间在 tidb_auto_analyze_start_time 和 tidb_auto_analyze_end_time 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句自动更新这个表的统计信息。
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
这个变量用来设置自动 ANALYZE 更新的阈值。当某个表 `tbl` 的修改行数与总行数的比值大于 tidb_auto_analyze_ratio，并且当前时间在 tidb_auto_analyze_start_time 和 tidb_auto_analyze_end_time 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句以自动更新该表的统计信息。
```

File: dev/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 2]
```diff
@@ -387,3 +411,13 @@ set @@global.tidb_distsql_scan_concurrency = 10
...
 
 可设置为 `NO_PRIORITY`、`LOW_PRIORITY`、`DELAYED` 或 `HIGH_PRIORITY`。
+
+### tidb_opt_write_row_id
+
+作用域：SESSION
+
+默认值：0
+
+这个变量用来设置是否允许 insert、replace 和 update 操作 _tidb_rowid 列，默认是不允许操作。该选项仅用于 TiDB 工具导数据时使用。
```
Comment (yikeke) [inline dev/reference/configuration/tidb-server/tidb-specific-variables.md]:
```suggestion
这个变量用来设置是否允许 insert、replace 和 update 操作 `_tidb_rowid` 列，默认是不允许操作。该选项仅用于 TiDB 工具导数据时使用。
```

File: dev/reference/sql/statements/ddl.md [hunk 3]
```diff
@@ -198,6 +198,18 @@ DROP TABLE [IF EXISTS]
 
 关键字 RESTRICT 和 CASCADE 没有实际效果。其作用是与其他数据库兼容。
 
+## RECOVER TABLE 语法
+
+```sql
+# 根据表名恢复删除的表。
+RECOVER TABLE tbl_name
+
+# 根据drop table 时的 DDL job ID 来恢复删除的表。
```
Comment (yikeke) [inline dev/reference/sql/statements/ddl.md]:
```suggestion
# 根据 drop table 时的 DDL job ID 来恢复删除的表。
```

File: dev/reference/sql/statements/ddl.md [hunk 4]
```diff
@@ -198,6 +198,18 @@ DROP TABLE [IF EXISTS]
 
 关键字 RESTRICT 和 CASCADE 没有实际效果。其作用是与其他数据库兼容。
 
+## RECOVER TABLE 语法
+
+```sql
+# 根据表名恢复删除的表。
```
Comment (yikeke) [inline dev/reference/sql/statements/ddl.md]:
```suggestion
# 根据表名来恢复删除的表。
```

File: dev/reference/sql/statements/ddl.md [hunk 5]
```diff
@@ -198,6 +198,18 @@ DROP TABLE [IF EXISTS]
...
+RECOVER TABLE tbl_name
+
+# 根据drop table 时的 DDL job ID 来恢复删除的表。
+RECOVER TABLE BY JOB job_id
+```
+
+用来恢复被删除的表。能够恢复的前提条件是还没有超过GC时间。因为超过 GC 时间数据被 GC 后，就无法恢复了。
```
Comment (yikeke) [inline dev/reference/sql/statements/ddl.md]:
```suggestion
用来恢复被删除的表。能够恢复的前提条件是还没有超过 GC 时间。因为超过 GC 时间后数据会被 GC，就无法恢复了。
```

**Comment by lilin90** [comment]
@crazycs520 Please address comments. Thanks!

File: dev/reference/configuration/tidb-server/tidb-specific-variables.md [hunk 6]
```diff
@@ -57,6 +57,30 @@ set @@global.tidb_distsql_scan_concurrency = 10
 
 这个变量用来设置优化器是否执行 `in-` 子查询展开的优化操作。
 
+### tidb_auto_analyze_ratio
+
+作用域: GLOBAL
+
+默认值: 0.5
+
+这个变量用来设置自动 ANALYZE 更新的阈值。当某个表 `tbl` 的修改行数与总行数的比值大于 tidb_auto_analyze_ratio，并且当前时间在 tidb_auto_analyze_start_time 和 tidb_auto_analyze_end_time 之间时，TiDB 会在后台执行 `ANALYZE TABLE tbl` 语句以自动更新该表的统计信息。
```
Comment (zimulala) [inline dev/reference/configuration/tidb-server/tidb-specific-variables.md]: 应该还需要开启 run-auto-analyze ？

Comment (crazycs520) [reply dev/reference/configuration/tidb-server/tidb-specific-variables.md]: Thanks, PTAL again.

**Pull request merged**